File: src/main/java/supercritical/api/registry/SCRegistries.java
package supercritical.api.registry;

import com.gregtechceu.gtceu.api.registry.registrate.GTRegistrate;
import supercritical.SCValues;

public final class SCRegistries {

    public static final GTRegistrate REGISTRATE = GTRegistrate.create(SCValues.MODID);

    private SCRegistries() {}
}

File: src/main/kotlin/supercritical/api/registries/SCRegistries.kt
package supercritical.api.registries;

import com.gregtechceu.gtceu.api.registry.registrate.GTRegistrate;
import org.jspecify.annotations.NullMarked;
import supercritical.BuildConfig;

@NullMarked
public final class SCRegistries {
    public static final GTRegistrate REGISTRATE = GTRegistrate.create(BuildConfig.MOD_ID);
}
